### PR TITLE
Add Serenity PCB to QMK

### PR DESCRIPTION
**Pull request by awkannan**
## Description

Add the PCB for the Serenity keyboard to QMK 

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/cannonkeys/serenity/info.json [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+        "pin": "A6"
+    },
+    "rgblight": {
+        "led_count": 12,
+        "pin": "B15",
+        "hue_steps": 24,
+        "saturation_steps": 16,
+        "brightness_steps": 16,
+        "animations": {
+            "all": true
```
Comment (drashna) [inline keyboards/cannonkeys/serenity/info.json:41]: Could you explicitly define which animations are enabled? 
```suggestion
            "alternating": true,
            "breathing": true,
            "christmas": true,
            "knight": true,
            "rainbow_mood": true,
            "rainbow_swirl": true,
            "rgb_test": true,
            "snake": true,
            "static_gradient": true,
            "twinkle": true
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/cannonkeys/serenity/info.json (hunk 1 above)
Comment (awkannan) [reply keyboards/cannonkeys/serenity/info.json:41]: I can do this - but as a note, wouldn't it be best to remove the "all" option if you don't want anyone to use it?

**Review by drashna — APPROVED** [review]

File: keyboards/cannonkeys/serenity/info.json (hunk 1 above)
Comment (drashna) [reply keyboards/cannonkeys/serenity/info.json]: Yeah, that's the plan for the next cycle.

File: keyboards/cannonkeys/serenity/readme.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# Serenity
+
+*A calming F13 TKL from HoodrowThrillson*
```
Comment (fauxpark) [inline keyboards/cannonkeys/serenity/readme.md:3]:
```suggestion
A calming F13 TKL from HoodrowThrillson
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Comment by awkannan** [comment]
@fauxpark Ready for re-review :) Thank you!

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Pull request merged**